src/passes/enumConverter.ts

```diff
@@ -34,6 +37,37 @@ export class EnumConverter extends ASTMapper {
     return val;
   }
 
+  visitFunctionCall(node: FunctionCall, ast: AST): void {
+    this.visitExpression(node, ast);
+    if (node.kind !== FunctionCallKind.TypeConversion) return;
+    const tNode = getNodeType(node.vExpression, ast.compilerVersion);
+    assert(
+      tNode instanceof TypeNameType,
+      `Got non-typename type ${tNode.pp()} when parsing conversion functiobin/warp transpile --print-trees tests/behaviour/solidity/test/libsolidity/semanticTests/enums/constructing_enums_from_ints.sol
```

Comment:
I think you must have accidentally pasted the command into this comment

Reply:
Yeah, Sorry for this.